pkg/build/builder/cmd/builder.go

```diff
@@ -131,7 +132,18 @@ func newBuilderConfigFromEnvironment(out io.Writer, needsDocker bool) (*builderC
 		}
 		istorage.Transport.SetStore(store)
 
-		dockerClient, err := bld.GetDaemonlessClient(systemContext, store, os.Getenv("BUILD_ISOLATION"))
+		// By default, enable a blob cache, but allow it to be moved or
+		// disabled via the environment.
+		cfg.blobCache = "/var/cache/blobs"
+		if dir, isSet := os.LookupEnv("BUILD_BLOBCACHE"); isSet {
```

Comment:
How will this be disabled - empty env valule?

Reply:
please call this BUILD_ENABLE_BLOBCACHE or something that makes it clear it's an enablement (and then make it enable when the value is set to `true` (case insensitive))

Reply:
Okay, changing it.

Reply:
sorry I screwed this up.  but i think BUILD_BLOBCACHE_DIR would be a better name.

Reply:
(and ignore my comments about expecting a value of `true`)